src/components/breadcrumbs/BreadcrumbList.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ * Echoes React
+ * Copyright (C) 2023-2025 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+import styled from '@emotion/styled';
+import React, { forwardRef } from 'react';
+import { isDefined } from '~common/helpers/types';
+import { IconSlash } from '../icons';
+import { BreadcrumbEllipsis, BreadcrumbEllipsisProps } from './BreadcrumbEllipsis';
+import { BreadcrumbLink, BreadcrumbLinkProps } from './BreadcrumbLink';
+
+export interface BreadcrumbListProps {
+  className?: string;
+  items: (BreadcrumbLinkProps | BreadcrumbEllipsisProps)[];
+}
+
+const isEllipsisItem = (
+  item: BreadcrumbLinkProps | BreadcrumbEllipsisProps,
+): item is BreadcrumbEllipsisProps => isDefined((item as BreadcrumbEllipsisProps).items);
+
+const BreadcrumbListBase = forwardRef<HTMLDivElement, BreadcrumbListProps>((props, ref) => {
+  const { items, ...rest } = props;
+
+  return (
+    <div ref={ref} {...rest}>
+      {items.map((item, index) => {
+        let breadcrumb;
+
+        if (isEllipsisItem(item)) {
+          breadcrumb = <BreadcrumbEllipsis {...item} />;
+        } else {
+          breadcrumb = (
+            <BreadcrumbLink
+              {...(index === items.length - 1 ? { ...item, isChild: true, to: '' } : { ...item })}
+            />
+          );
+        }
+
+        return (
+          <React.Fragment key={crypto.randomUUID()}>
+            {Boolean(index) && <IconSlash color="echoes-color-icon-subdued" />}
+
+            {breadcrumb}
+          </React.Fragment>
+        );
+      })}
+    </div>
+  );
+});
+
+BreadcrumbListBase.displayName = 'BreadcrumbListBase';
+
+export const BreadcrumbList = styled(BreadcrumbListBase)`
+  display: flex;
+  flex-wrap: wrap;
+  font-size: var(--echoes-font-size-10);
```

Comment:
Let's use a full typography design token instead of `font-size` and `line-height` separatly.